uvicorn/protocols/http/h11_impl.py

```diff
@@ -412,6 +413,8 @@ async def run_asgi(self, app: "ASGI3Application") -> None:
             result = await app(  # type: ignore[func-returns-value]
                 self.scope, self.receive, self.send
             )
+        except ClientDisconnected:
+            ...
```

Comment:
```suggestion
            pass
```

I think is more common in this context?